perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -588,6 +588,13 @@ sub property : Chained('/') : PathPart('waste') : CaptureArgs(1) {
         $c->detach;
     }
 
+    # The calendar help page doesn't need anything but the property ID,
```

Comment:
I think a clearer way to do this would be to stop calendar being Chained to property entirely - if it was its own route then you wouldn't need to complicate this with a special case. ie `sub calendar : Path : Args(1) {` or similar, then sticking the ID on the stash.

Reply:
I did try this to begin with but couldn't figure out the right incantation - will have another go based off your suggestion 👍

Reply:
Have moved to `LocalRegex` in a22a4007bb48d38018883786108d9ce7991c77d2, though that seems to be deprecated and the docs suggest using chains instead. But I can't see a way to do that without chaining off `property`...

Reply:
Ugh, sorry, I must have had it in my head it was calendar/ID... So this way is fine (we have a number of other LocalRegexes), or your original way - I guess the actual cleanest would be to change `property` to Chain off a new tiny function (property_id?) that fetched the ID and didn't do anything else, and then calendar could Chain off of property_id, leaving rest of property alone.

Reply:
Have pushed a change to use Chain for the ID alone, let me know if that works for you

Reply:
Looks great, thanks. Happy for me to merge?

Reply:
Yep!

Reply:
Actually, need to deploy, so doing it now.